jsonschema2dj/valdiators.py

```diff
@@ -43,3 +43,18 @@ def __call__(self, payload):
                     for error in errors
                 ]
             )
+
+
+class MultipleOfValidator(validators.BaseValidator):
+    message = _('%(value)s is not a multiple of %(factor)')
+    code = 'factor'
+
+    def __init__(self, factor, message=None):
+        self.factor = factor
+        if message:
+            self.message = message
```

Comment:
message doesnt do anything?
could either remove, or if you want to keep it set like:

```python
    def __init__(self, factor, message=None):
        self.factor = factor
        self.message = message
```

as its a good idea to keep the shape of the class fixed